test/system/060-mount.bats

```diff
@@ -148,6 +148,14 @@ load helpers
     run_podman run --rm --mount "${mountopts},readwrite=true" $iname \
                touch /image-mount/readwrite
 
+    tmpctr="c-$(safename)"
+    subpathopts="type=image,src=$iname,dst=/image-mount,subpath=/etc"
+    run_podman run --name $tmpctr --mount "${subpathopts},readwrite=true" $iname \
```

Comment:
What is the point of passing readwrite when we only read it

Reply:
Cut and paste from other mount, I can change if it  matters.  readWrite=true on a read-only image probably should throw an error, but not something I am volunteering on this PR.